cmd/kubernetes_create.go

```diff
@@ -33,9 +35,7 @@ Notes:
 * So the following would all be valid:
     * "80,443,6443:0.0.0.0/0;8080:1.2.3.4" (open 80,443,6443 to public and 8080 just for 1.2.3.4/32)
     * "80,443,6443;6000-6500:4.4.4.4/24" (open 80,443,6443 to public and 6000 to 6500 just for 4.4.4.4/24)
-* When '--create-firewall' flag is blank, your cluster will be created with the following rules:
-    * "80;443;6443" (open 80,443,6443 to public)
-* To open all ports for public access, "all" can be provided to '--create-firewall' flag (not recommended)
+* When '--create-firewall' flag is blank, your cluster will be created with the default rules all open
```

Comment:
see comments above. We seem to be using "default" and "all open" and somewhere even 80, 443, 6443 - Should be consistent throughout.

Reply:
I remove this line because the first one say the same:
`* The '--create-firewall' will add default rules for the firewall if none are specified by using '--firewall-rules'.`